app/controllers/users.js

```diff
@@ -3,19 +3,22 @@
  */
 var mongoose = require('mongoose'),
```

Comment:
1. It is advisable to use `const` throughout and be consistent with it.

2. I will suggest you define and end each line of expression.

Example: Instead of
 ``` 
const a = require('a'),
     b = require('b');
  ```
Use: 
``` 
      const a = require ('a');
      const b = require('b');
```
Reason: Ease to read and debug.